docs/sphinx/specification/quake-dialect.md

```diff
@@ -27,7 +27,7 @@ greater, the extra qubits are considered controls.
 
 The main motivation behind Quake's value model is to directly expose
 quantum and classical data dependencies for optimization purposes,
-i.e., to represent the dataflow in quantum computations.  In contrast
+i.e., to represent the data flow in quantum computations.  In contrast
```

Comment:
dataflow is a legit single word. https://www.merriam-webster.com/dictionary/dataflow

Reply:
Updated in bbd1449b6.